core/tests/protractor_utils/editor.js

```diff
@@ -397,6 +397,30 @@ var addParameterChange = function(paramName, paramValue) {
   general.waitForSystem(500);
 };
 
+// This function adds a exploration level parameter change, creating
+// the parameter if necessary.
+var addExplorationLevelParameterChange = function(paramName, paramValue) {
+  runFromSettingsTab(function() {
+    element(by.css('.protractor-test-exploration-edit-param-changes')).click();
+    element(by.css('.protractor-test-add-param-button')).click();
+
+    var editorRowElem = element.all(by.css(
+    '.protractor-test-param-changes-list')).last();
+
+    forms.AutocompleteDropdownEditor(editorRowElem).setValue(paramName);
+
+    /* Setting parameter value is difficult via css since the associated
+    input is a sub-component of the third party select2 library. We isolate
```

Comment:
Are you sure about this? The parameter name is selected using a select2 dropdown, but the value is a standard input field, I think.

That said, this may be a reasonable approach anyway. I'll leave it to @arun1595 to make the final ruling (though you may still want to change the comment to reflect reality, if necessary).